src/ld65/config.c

```diff
@@ -543,6 +545,8 @@ static void ParseFiles (void)
         {   "FORMAT",   CFGTOK_FORMAT   },
     };
     static const IdentTok Formats [] = {
+        {   "ATARI",    CFGTOK_XEX      },
+        {   "XEX",      CFGTOK_XEX      },
```

Comment:
Is "atari" format an alias for "xex"? If yes, I'd prefer to get rid of "xex" inside the cfg file (and source code) and just use "atari", or "atariexe". The *.cfg file names can still contain "xex" if you prefer. It's a handler for "Atari EXE" file format.